schema/bom-1.5.proto

```diff
@@ -675,3 +677,29 @@ enum VulnerabilityAffectedStatus {
   VULNERABILITY_AFFECTED_STATUS_AFFECTED = 1;
   VULNERABILITY_AFFECTED_STATUS_NOT_AFFECTED = 2;
 }
+
+message AnnotatorChoice {
+  oneof choice {
+    // The organization that created the annotation
+    OrganizationalEntity organization = 1;
+    // The person that created the annotation
+    OrganizationalContact individual = 2;
+    // The tool or component that created the annotation
+    Component component = 3;
+    // The service that created the annotation
+    Service service = 4;
+  }
+}
+
+message Annotation {
+  // An optional identifier which can be used to reference the annotation elsewhere in the BOM. Every bom-ref MUST be unique within the BOM.
+  optional string bom_ref = 1;
+  // The object in the BOM identified by its bom-ref. This is often a component or service, but may be any object type supporting bom-refs.
+  repeated string subjects = 2;
+  // The organization, person, component, or service which created the textual content of the annotation.
+  AnnotatorChoice annotator = 3;
+  // The date and time (timestamp) when the annotation was created.
+  google.protobuf.Timestamp timestamp = 4;
+  // The textual content of the annotation.
+  string text = 5;
```

Comment:
Is the exclusion of "signature" because Protocol Buffers don't really need it?

Reply:
I think because there isn't, as far as I know, a good way to sign parts of a protobuf message

Reply:
Correct. I'm not aware of any solution for signing in protobuf.